modules/deleting-service.adoc

```diff
@@ -4,28 +4,27 @@
 // * assemblies/adding-service.adoc
 
 [id="deleting-service_{context}"]
+= Deleting a service from a cluster
 
-= Deleting a service from your {product-title} cluster
 
-
-You can delete add-on services from your {product-title} cluster through the OpenShift Cluster Manager (OCM) or the OCM CLI.
+You can delete the add-on services from your {product-title} cluster through the OpenShift Cluster Manager (OCM) console or the OCM CLI.
 
 .Procedure
 
 . Navigate to the *Clusters* page in link:https://cloud.redhat.com/openshift/[OCM].
 
-. Select the cluster with the installed service you want to delete.
+. Click the cluster with the installed service that you want to delete.
 
-. Navigate to the *Add-ons* tab, and locate the installed service you want to delete.
+. Navigate to the *Add-ons* tab, and locate the installed service that you want to delete.
 
-. From the installed service option, click the menu and select *Uninstall add-on* from the drop down menu.
+. From the installed service option, click the menu and select *Uninstall add-on* from the drop-down menu.
 
-. You must type the name of the service you want to delete in the confirmation message that appears.
+. You must type the name of the service that you want to delete in the confirmation message that appears.
 
-. Click *Uninstall*. You are returned to the *Add-ons* tab and an uninstalling state icon will be present on the service option you deleted.
+. Click *Uninstall*. You are returned to the *Add-ons* tab and an uninstalling state icon is present on the service option you deleted.
 
 . To delete the add-on service from your cluster through the OCM CLI, enter the following command:
 +
 ----
```

Comment:
I'm sorry I missed this the first go-around, but this code block is missing the `[source,terminal]`.